src/interval_sets.jl

```diff
@@ -469,16 +469,15 @@ function intersection_isless_fn(::TrackEachEndpoint)
 end
 
 """
-    find_intersections(
-        x::AbstractVector{<:AbstractInterval},
-        y::AbstractVector{<:AbstractInterval}
-    )
+    find_intersections(x, y)
```

Comment:
The docstring should go into more detail about acceptable arguments to this function now that context from the types is no longer available